package.json

```diff
@@ -160,6 +160,7 @@
     "vue-property-decorator": "7.0.0",
     "vue-slider-component": "2.2.7",
     "vue-template-compiler": "2.5.16",
+    "vue-toasted": "^1.1.24",
```

Comment:
It looks like this is in here twice now.  It got added out of alphabetical order at the end of the list.  Can you delete the one at the end of the list?